scripts/qa_crowbarsetup.sh

```diff
@@ -3078,7 +3078,6 @@ function onadmin_testsetup()
 
     cephret=0
     if [ -n "$deployceph" -a "$wantcephtestsuite" == 1 ] ; then
-        ensure_packages_installed git-core
```

Comment:
I would not remove this line. Because via this function call this code block states its dependecies.
When the code changes that runs earlier, then git might be missing. So the correct way is to call the `ensure_packages_installed` function in both places.